azure-spring-boot-tests/azure-spring-boot-test-aad/src/test/java/com/microsoft/azure/test/keyvault/group/AADAuthenticationFilterIT.java

```diff
@@ -48,14 +50,25 @@
     private RestTemplate restTemplate = new RestTemplate();
 
     @Test
-    public void testAADAuthenticationFilter() {
-        final OAuthResponse authResponse = OAuthUtils.executeOAuth2ROPCFlow();
+    public void testAADAuthenticationFilterWithSingleTenantApp() {
+        testAADAuthenticationFilter(System.getenv(SINGLE_TENANT_AAD_CLIENT_ID),
+                System.getenv(SINGLE_TENANT_AAD_CLIENT_SECRET));
+    }
+
+    @Test
+    public void testAADAuthenticationFilterWithMultiTenantApp() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/Microsoft/azure-spring-boot/pullRequest?prid=5157494)